src/test/java/com/wikia/webdriver/common/core/CommonUtils.java

```diff
@@ -74,6 +74,14 @@ public static void createDirectory(String fileName) {
     }
   }
 
+  public static String getAbsolutePathForFile(String relativePath) {
+    File fileCheck = new File(relativePath);
+    if (!fileCheck.isFile()) {
+      throw new RuntimeException("file " + relativePath + " doesn't exists");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)